Implementation/src/Logger/Log.cpp

```diff
@@ -34,6 +36,7 @@ void Logger::setEnabledLogs(const bool enabled)
 
 void Logger::log(const std::string& msg)
 {
+    if (CERRlOGS) std::cerr << msg << std::endl;
```

Comment:
This is useful for debugging, but writing to std::cerr could impact performance in production. Consider adding a runtime flag to toggle this behavior.